src/Dfc.CourseDirectory.Database/Pttcd/Tables/FeChoices.sql

```diff
@@ -0,0 +1,8 @@
+﻿ CREATE TABLE [Pttcd].[FeChoices]
```

Comment:
As above, suggest adding this to `Providers`

Reply:
removed now.